client/extensions/woocommerce/state/sites/payment-methods/test/selectors.js

```diff
@@ -97,41 +97,20 @@ describe( 'selectors', () => {
 		} );
 	} );
 
-	describe( '#getPaymentMethodsGroup', () => {
-		it( 'should be an empty array when woocommerce state is not available.', () => {
-			expect( getPaymentMethodsGroup( preInitializedState, 'offline', 123 ) ).to.be.empty;
+	describe( '#getApiPaymentMethods', () => {
+		it( 'should return paymentMethods when given populated state tree.', () => {
+			expect( getApiPaymentMethods( loadedState, 123 ) )
+				.to.deep.equal(
+					loadedState.extensions.woocommerce.sites[ 123 ].paymentMethods
+				);
 		} );
 
-		it( 'should be an empty array when there are no methods matching this type.', () => {
-			expect( getPaymentMethodsGroup( loadedState, 'nonexistent', 123 ) ).to.be.empty;
+		it( 'should return LOADING constant when given a loading state tree.', () => {
+			expect( getApiPaymentMethods( loadingState, 123 ) ).to.deep.equal( 'LOADING' );
```

Comment:
You can also use `.eql()`, i believe it does the same as `.deep.equal()` when you don't have more in the chain. (more an "FYI" than a request to change it)